engines/e_afalg.c

```diff
@@ -699,13 +723,15 @@ static int bind_afalg(ENGINE *e)
     }
 
     /*
-     * Create _hidden_aes_128_cbc by calling afalg_aes_128_cbc
+     * Create _hidden_aes_xxx_cbc by calling afalg_aes_xxx_cbc
      * now, as bind_aflag can only be called by one thread at a
      * time.
      */
-    if (afalg_aes_128_cbc() == NULL) {
-        AFALGerr(AFALG_F_BIND_AFALG, AFALG_R_INIT_FAILED);
-        return 0;
+    for(i=0; i < sizeof(afalg_cipher_nids)/sizeof(afalg_cipher_nids[0]);i++) {
```

Comment:
Use the OSSL_NELEM macro for finding the size of an array. Also, style nit: please insert spaces around the '=' (i.e. `i = 0`), and after the second ;